frontend/public/components/_secret.scss

```diff
@@ -0,0 +1,29 @@
+.btn-file {
```

Comment:
So in case we want the input field with `Browse` btn for picking file, we need additional styling since the Bootstrap does not offer this styling by default.
Check origin-web-console [styling](https://github.com/openshift/origin-web-console/blob/master/app/styles/_buttons.less#L3-L19) and [usage](https://github.com/openshift/origin-web-console/blob/master/app/views/directives/osc-file-input.html#L5-L20). Also was browsing similar use-cases and more less this approach is pretty common.